runtime/src/main/scala/akka/grpc/GrpcClientSettings.scala

```diff
@@ -95,6 +95,13 @@ object GrpcClientSettings {
         if (serviceName.isEmpty)
           serviceName = "static"
         staticServiceDiscovery(host, port)
+      case "grpc-dns" =>
```

Comment:
So there is currently no difference between `static` and `grpc-dns`? Perhaps we could use `case "static" | "grpc-dns"` to make that clearer?

(it is a bit strange to introduce a `grpc-dns` discovery method in the config, but actually have a 'static' implementation for it instead of having a 'real' `grpc-dns` discovery implementation.

I think this is acceptable as long as we're using the netty client implementation, so this is a special-case delegating the discovery to that. Eventually when we move from netty to akka-http for the client this would probably change, but that's OK)

Reply:
Indeed thats a bit strange, I named service discovery mechanism `grpc-dns` to indicate that we are delegating actual discovery to grpc internals, not ideal but had no better idea...